src/pages/AccountPage.vue

```diff
@@ -155,6 +156,13 @@ async function loadAccount() {
                         :alt="contract.getName() + ' ERC20 token'"
                         :src="getIcon(contract.logoURI)"
                     />
+                    <img
+                        v-else
+                        :src="createIconFromData(accountAddress)"
+                        alt=""
```

Comment:
alt="Blockies icon for address {{ accountAddress }}" or something similar would work here right?